### DISCUSSION TITLE
Can I see a list of which repositories I have indexed in GitHub Copilot?

### DISCUSSION THREAD

**Discussion by soumyadas-09** [community/community#144914] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

My GitHub Copilot Individual account enables me to [index 5 repositories](https://docs.github.com/en/copilot/managing-copilot/managing-github-copilot-in-your-organization/customizing-copilot-for-your-organization/indexing-repositories-for-copilot-chat#indexing-limits).

I have searched the UI and help docs, but can't find a list of those I have chosen to index so far.

Any idea where I can find my currently indexed repository list?

**Comment by SubhashPavuskar** [comment, marked as answer]
Even with Enterprise sku I still cannot find where repositories are indexed, when you reach limit, you can either delete the repository (sound awkward tbh), or ask other users to index it for you - you still can benefit that

Indexing is permanent for each repository unless the repository itself is deleted. Use caution when selecting a repository to index. Repository indexing cannot be canceled once it starts.

If you have a Copilot Individual (CI) plan, you are subject to an indexing limit and can index any repository that you have access to. However, you can still benefit from the indexes created by other users without affecting your limit.

https://docs.github.com/en/copilot/customizing-copilot/indexing-repositories-for-copilot-chat#indexing-limits
